lnrpc/signrpc/signer_server.go

```diff
@@ -657,6 +687,38 @@ func (s *Server) VerifyMessage(ctx context.Context,
 	if in.Pubkey == nil {
 		return nil, fmt.Errorf("a pubkey to verify MUST be passed in")
 	}
+
+	// We allow for Schnorr signatures to be verified.
+	if in.IsSchnorrSig {
+		var keyCompressed [btcec.PubKeyBytesLenCompressed]byte
```

Comment:
You can just use `schnorr.ParsePubKey()` for 32-byte public keys.

Reply:
@guggero My rationale was to not have to import the "github.com/decred/dcrd/dcrec/secp256k1/v4" package. It's used in btcd so maybe it doesn't matter.

Reply:
Great